### ISSUE TITLE
AVT 1 - React Toolbar has DAP violations for missing labels.

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3376]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP - July 2019 Ruleset

## Detailed Description
Run DAP on the Toolbar component and the following violations are present:
<img width="1060" alt="Screen Shot 2019-07-11 at 5 01 45 PM" src="https://user-images.githubusercontent.com/21676914/61088335-c8a7bc00-a3fd-11e9-8005-3b123921748e.png">
